x-pack/plugins/security_solution/public/overview/components/host_overview/index.tsx

```diff
@@ -49,8 +50,12 @@ interface HostSummaryProps {
   narrowDateRange: NarrowDateRange;
 }
 
-const getDescriptionList = (descriptionList: DescriptionList[], key: number) => (
-  <EuiFlexItem key={key}>
+const getDescriptionList = (
```

Comment:
nit -> I know you did not write that but I am wondering if we can move `getDescriptionList` with `DescriptionListStyled` so we do not duplicate the same component twice.
I have the same nit that on `x-pack/plugins/security_solution/public/overview/components/host_overview/index.tsx`

Reply:
Yea, wasn't sure what the pattern was as the file that has `DescriptionListStyled` seemed to just be a file of styles, so I just created `OverviewDescription` to reuse the component between the Network and Hosts pages